.gitlab-ci.yml

```diff
@@ -198,6 +198,7 @@ test-lin-dev-clang-cran:
     - echo 'CFLAGS=-g -O2 -fno-common -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' > ~/.R/Makevars
     - echo 'CXXFLAGS=-g -O2 -fno-common -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' >> ~/.R/Makevars
     - *install-deps
+    - clang-tidy -extra-arg=-I/usr/local/lib/R/include -checks='readability-inconsistent-declaration-parameter' src/*.c -- -std=c99
```

Comment:
Comments describing this job, here in the file, could mention about this check as well. Also .ci/README.MD could be updated for that.

Reply:
Updating the .ci/README.md and adding some best practices like developing CI changes on Gitlab and then fetching them here is on my long standing to-do list